batchflow/models/metrics/loss.py

```diff
@@ -0,0 +1,44 @@
+"""  Loss as a Metrics to be used in research pipelines added with `run=True` """
+
+import numpy as np
+
+from .base import Metrics
+
+
+class Loss(Metrics):
```

Comment:
Can we use `RegressionMetrics` instead of the new `Loss` class?

Reply:
We can't use any existing metric because they all calculate needed values from given predictions and targets, and loss is already calculated during prediction. 

We also have to keep batch lengths along with average loss on each batch to compute true average loss on the whole dataset. These are the motivations for a new class.